src/components/LanguageDropdown/LanguageDropdown.css

```diff
@@ -21,12 +21,16 @@
 .languageDropdown .languageSelect .languageSelectContainer {
   display: grid;
   grid-template-columns: 40px auto;
-  width: 135px;
-  background-color: rgb(53, 106, 218);
+  width: 155px;
+  background-color: #2A4EF5;
   color: white;
-  border-radius: 3px;
+  border-radius: 8px;
 }
 
+/* .languageDropdown .languageSelect .languageSelectContainer  {
```

Comment:
lets remove commented out code no need to leave it in since we have source control